docs/shrc_diag_util.md

```diff
@@ -262,106 +282,170 @@ Line number information for classes in the cache is then shown with one of the f
 
 The following summary data is displayed:
 
-#### `baseAddress` and `endAddress`
- : The boundary addresses of the shared memory area that contains the classes.
-
-#### `allocation pointer`
- : The address where `ROMClass` data is currently being allocated in the cache.
-
-#### `cache layer`
-: The layer number that the cache stats relate to.
-
-#### `cache size` and `free bytes`
- : `cache size` shows the total size of the shared memory area in bytes, and `free bytes` shows the free bytes that remain.
+- `baseAddress` and `endAddress`
+    - The boundary addresses of the shared memory area that contains the classes.
+- `allocation pointer`
+    - The address where `ROMClass` data is currently being allocated in the cache.
+- `cache layer`
+    - The layer number that the cache stats relate to.
+- `cache size` and `free bytes`
+    - `cache size` shows the total size of the shared memory area in bytes
+    - `free bytes` shows the free bytes that remain.
+- `softmx bytes`
+    - The soft maximum size for the cache. For more information, see [`-Xscmx`](xscmx.md).
+- `ROMClass bytes`
+    - The number of bytes of class data in the cache.
+- `AOT bytes`
+    - The number of bytes of AOT-compiled code in the cache.
+- `Reserved space for AOT bytes`
+    - The number of bytes reserved for AOT-compiled code in the cache.
+- `Maximum space for AOT bytes`
+    - The maximum number of bytes of AOT-compiled code that can be stored in the cache.
+- `JIT data bytes`
+    - The number of bytes of JIT-related data stored in the cache.
+- `Reserved space for JIT data bytes`
+    - The number of bytes reserved for JIT-related data in the cache.
+- `Maximum space for JIT data bytes`
+    - The maximum number of bytes of JIT-related data that can be stored in the cache.
+- `Zip cache bytes`
+    - The number of zip entry cache bytes stored in the cache.
```

Comment:
This is expected to be zero for Java 11 and above, unless a jar file is added to the bootclasspath.